### ISSUE TITLE
[Bug]: Add Support for Qt 6.8.x [win64_msvc2022_64] Binaries

### ISSUE THREAD

**Issue by xavier2k6** [miurahr/aqtinstall#796]
### Bug description

Can't install Qt 6.8.0 Beta1 [available here](https://download.qt.io/online/qtsdkrepository/windows_x86/desktop/qt6_680/qt6_680/) as [announced here](https://www.qt.io/blog/qt-6.8-beta-1-released)

Qt have moved to MSVC 2022 Packages as of Qt 6.8 as [announced here](https://www.qt.io/blog/moving-to-msvc-2022-in-qt-68)

### Expected behavior

To be able to install/test Qt 6.8.x

### aqt and python version

aqtinstall(aqt) v3.1.15 on Python 3.11.9 [CPython MSC v.1938 64 bit (AMD64)]

### Operating System

Windows

### Relevant log output

```shell
C:\hostedtoolcache\windows\Python\3.11.9\x64\python.exe -m aqt install-qt windows desktop 6.8.0 win64_msvc2019_64 --autodesktop --outputdir D:\a\qBittorrent/Qt --archives qtbase qtsvg qttools
```

```
INFO    : aqtinstall(aqt) v3.1.15 on Python 3.11.9 [CPython MSC v.1938 64 bit (AMD64)]
WARNING : Specified Qt version "6.8.0" did not exist when this version of aqtinstall was released. This may not install properly, but we will try our best.
WARNING : Failed to download checksum for the file 'Updates.xml'. This may happen on unofficial mirrors.
ERROR   : Failed to locate XML data for Qt version '6.8.0'.
```
____

```
INFO    : aqtinstall(aqt) v3.1.15 on Python 3.11.9 [CPython MSC v.1938 64 bit (AMD64)]
C:\hostedtoolcache\windows\Python\3.11.9\x64\python.exe -m aqt install-qt windows desktop 6.8.0 win64_msvc2022_64 --autodesktop --outputdir D:\a\qBittorrent/Qt --archives qtbase qtsvg qttools
INFO    : aqtinstall(aqt) v3.1.15 on Python 3.11.9 [CPython MSC v.1938 64 bit (AMD64)]
WARNING : Specified Qt version "6.8.0" did not exist when this version of aqtinstall was released. This may not install properly, but we will try our best.
WARNING : Specified target combination "windows desktop win64_msvc2022_64" did not exist when this version of aqtinstall was released. This may not install properly, but we will try our best.
WARNING : Failed to download checksum for the file 'Updates.xml'. This may happen on unofficial mirrors.
ERROR   : Failed to locate XML data for Qt version '6.8.0'.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by miurahr** [comment]
Here is a path for arm64 version
```
- https://download.qt.io/online/qtsdkrepository/windows_x86/desktop/qt6_663/qt.qt6.663.win64_msvc2019_arm64/
- https://download.qt.io/online/qtsdkrepository/windows_arm64/desktop/qt6_680/qt6_680/qt.qt6.680.win64_msvc2022_arm64/
```

**Comment by miurahr** [comment]
aqt expect package name as `qt.qt6.680.msvc22_x86` but `Update.xml` has `qt.qt6.680.win64_msvc2022_64`

**Comment by xavier2k6** [comment]
@miurahr Note: Qt 6.7.x will have **both** MSVC2019 & MSVC2022 binaries/packages (possibly `.3` release, as `.2` is just out) where as Qt 6.8.x will only have MSVC2022.

>In one of the next bug-fix releases of Qt 6.7, possibly in Qt 6.7.2, we will add packages built with MSVC 2022 in addition to the MSVC 2019 ones as we have now.

https://www.qt.io/blog/moving-to-msvc-2022-in-qt-68

**Comment by miurahr** [comment]
> @miurahr Note: Qt 6.7.x will have **both** MSVC2019 & MSVC2022 binaries/packages (possibly `.3` release, as `.2` is just out) where as Qt 6.8.x will only have MSVC2022.

You can specify  "win64_msvc2022_64" or "win64_msvc2019_64" as arch argument.  
https://aqtinstall.readthedocs.io/en/latest/cli.html#install-qt-command

If you have a trouble, please raise another issue ticket.
